src/js/app.js

```diff
@@ -29,6 +29,17 @@ const app = {
         $('#todos').append($todoContainer);
         
         app.counter = app.counter + 1;
+
+        $todoCheckbox.click(function () {
+
+            if(this.checked == true){
```

Comment:
En las condiciones, se considera una buena práctica utilizar operadores de comparación estricta (`===`) para evitar problemas con el [`type coercion`](https://stackoverflow.com/questions/19915688/what-exactly-is-type-coercion-in-javascript). Además en valores booleanos, no es necesario compararlos contra `true` o `false`.